sample/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,7 @@
+<resources>
+    <string name="app_name">ButtonKit Sample</string>
```

Comment:
Let's shorten sample app name to just "ButtonKit" so it doesn't truncate in the launcher app :)